WhaleSpotting/react-app/src/styles/Navbar.scss

```diff
@@ -0,0 +1,68 @@
+@import './constants.scss';
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+.header {
+    position: sticky;
+    top: 0;
+    width: 100%;
+    background-color: $primaryColour;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $headerHeight;
+    z-index: 10;
```

Comment:
Why was this added?

Reply:
Just to maintain a constant size across site.

Reply:
z-index doesn't do that, it changes whether it should be on top or bottom of a view, we won't need it yet